### ISSUE TITLE
Is there a way to finetune my own task? 

### ISSUE THREAD

**Issue by 52THANOS** [Parskatt/DeDoDe#8]
i want to use it in medical field

**Comment by Parskatt** [comment]
What type of supervision do you have?

**Comment by 52THANOS** [comment]
i wanna try fully supervised first

> What type of supervision do you have?

i wanna try fully supervised first

**Comment by Parskatt** [comment]
Sure, just load the pretrained weights and set up a cross entropy loss between your supervised detections and DeDoDe. Your detections should be an images with ones at detections and 0s otherwise.

**Comment by Parskatt** [comment]
Ill upload the training scripts of dedode later this week, then youll be able to see the setup.

**Comment by 52THANOS** [comment]
> Ill upload the training scripts of dedode later this week, then youll be able to see the setup

Lifesaver!

**Comment by Parskatt** [comment]
@52THANOS now there are reference training scripts in the [experiments](experiments) folder, hope that helps. Let me know if something is unclear.
